app/controllers/animals_controller.rb

```diff
@@ -0,0 +1,12 @@
+class AnimalsController < ApplicationController
+  def index
+
+  end
+  def show
+
+  end
+ def new
```

Comment:
If we have a new action we also need an create action, and we need to fix the identation of new action.